### PR TITLE
LoanApp - JDBC

### PR DESCRIPTION
**Pull request by umiidi**

### CODE REVIEW

File: LoanAppJDBC/src/main/java/az/growlab/LoanApp/model/Product.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+import lombok.Data;
+
+import java.math.BigDecimal;
+
+@Data
+public class Product {
+
+    private int id;
+    private int loan_info_id;
```
Comment (brhmshrnv) [inline LoanAppJDBC/src/main/java/az/growlab/LoanApp/model/Product.java]: use camelCase

File: LoanAppJDBC/src/main/java/az/growlab/LoanApp/model/AddressInformation.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package az.growlab.LoanApp.model;
+
+import lombok.Data;
+
+@Data
+public class AddressInformation {
+
+    private int id;
+    private String Country;
```
Comment (brhmshrnv) [inline LoanAppJDBC/src/main/java/az/growlab/LoanApp/model/AddressInformation.java]: country

File: LoanAppJDBC/src/main/java/az/growlab/LoanApp/model/AddressInformation.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    private String City;
```
Comment (brhmshrnv) [inline LoanAppJDBC/src/main/java/az/growlab/LoanApp/model/AddressInformation.java]: city

File: LoanAppJDBC/src/main/java/az/growlab/LoanApp/exception/ReasonNotFoundException.java [hunk 4]
```diff
@@ -0,0 +1,9 @@
+package az.growlab.LoanApp.exception;
+
+import org.webjars.NotFoundException;
```
Comment (brhmshrnv) [inline LoanAppJDBC/src/main/java/az/growlab/LoanApp/exception/ReasonNotFoundException.java]: are you sure about this import ?

File: LoanAppJDBC/src/main/java/az/growlab/LoanApp/exception/GlobalExceptionHandler.java [hunk 5]
```diff
@@ -0,0 +1,25 @@
+package az.growlab.LoanApp.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.webjars.NotFoundException;
```
Comment (brhmshrnv) [inline LoanAppJDBC/src/main/java/az/growlab/LoanApp/exception/GlobalExceptionHandler.java]: ?

File: LoanAppJDBC/src/main/java/az/growlab/LoanApp/enums/ConfirmStatus.java [hunk 6]
```diff
@@ -0,0 +1,9 @@
+package az.growlab.LoanApp.enums;
+
+public enum ConfirmStatus {
+
+    APPROVE,
+    NEED_TO_IMPROVE_DEVLINE,
```
Comment (brhmshrnv) [inline LoanAppJDBC/src/main/java/az/growlab/LoanApp/enums/ConfirmStatus.java]: NEED_TO_IMPROVE

**Review by brhmshrnv — CHANGES_REQUESTED** [review]

File: LoanAppJDBC/src/main/java/az/growlab/LoanApp/exception/ReasonNotFoundException.java (hunk 4 above)
Comment (umiidi) [reply LoanAppJDBC/src/main/java/az/growlab/LoanApp/exception/ReasonNotFoundException.java]: i think java offers this

Comment (brhmshrnv) [reply LoanAppJDBC/src/main/java/az/growlab/LoanApp/exception/ReasonNotFoundException.java]: better to extends RuntimeException

**Review by brhmshrnv — CHANGES_REQUESTED** [review]

File: LoanAppJDBC/src/main/java/az/growlab/LoanApp/exception/ReasonNotFoundException.java (hunk 4 above)
Comment (umiidi) [reply LoanAppJDBC/src/main/java/az/growlab/LoanApp/exception/ReasonNotFoundException.java]: okey, understand

**Pull request merged**